src/core/providers/helpers/walletconnect/components/WalletconnectModalComponent.ts

```diff
@@ -0,0 +1,87 @@
+import { LitElement, html, css } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import QRCode from 'qrcode';
+
+@customElement('wallet-connect-modal')
+export class WalletconnectModalComponent extends LitElement {
+  @property({ type: Boolean }) isOpen = false;
+  @property({ type: String }) qrCodeData = '';
+
+  static styles = css`
```

Comment:
is it mandatory to pe statically defined here? Do you think it could be better to extract in a separate file ?